src/App.js

```diff
@@ -17,14 +18,16 @@ class App extends Component {
         <div>
           <NavigationComponent />
           <Switch>
-            <Route path="/" component={HomeView} exact />
             <Route path="/login" component={LoginView} exact />
             <Route path="/register" component={RegisterView} />
             <Route path="/verify/:token" component={VerifyRegistrationView} />
             <Route path="/resetpassword" component={ResetPasswordView} />
             <Route path="/reset_password/:token" component={NewPasswordView} />
+            <Route path="/" component={ArticlesView} exact />
+            <Route path="/login" component={LoginView} exact />
```

Comment:
This Login route is a duplicate of the one above.

Reply:
Thanks for the observation. I have fixed this